src/apps/chifra/pkg/types/types_test.go

```diff
@@ -12,19 +12,21 @@ func TestTypes(t *testing.T) {
 		BlockNumber: 100,
 	}
 
-	if r.Model(false, "csv").Data["blockHash"] != nil {
+	data := r.Model(false, "csv", nil).Data.(map[string]interface{})
+
+	if data["blockHash"] != nil {
 		t.Fatal("hidden field leaked")
 	}
 
-	if r.Model(false, "csv").Data["blockNumber"] == nil {
+	if data["blockNumber"] == nil {
 		t.Fatal("visible field missing")
 	}
 
-	if r.Model(true, "json").Data["blockHash"] == nil {
+	if data["blockHash"] == nil {
```

Comment:
This test fails. Can we use this as a learning opportunity? Is this related to the conversion to `any` type?

Reply:
No, it was my mistake. This part of the test was calling `r.Model(true, "json")` before to force showing the hidden fields. I forgot to change these two arguments